src/components/Article/index.tsx

```diff
@@ -11,8 +11,13 @@ export const Article = ({ article, onCommentsClick }: ArticleProps) => {
   return (
     <div className="h-dyn-screen snap-center flex flex-col justify-end items-center relative max-w-[500px] mx-auto">
       <Image article={article} />
-      <div className="from-black/50 to-transparent bg-gradient-to-t p-3 pb-8 max-w-screen">
-        <div className="w-[calc(100%-0.75rem)] max-w-screen grid grid-cols-[90%_10%] gap-3 box-">
+      <div className={`
+          ${article.cover_image && "justify-end"}
+          ${!article.cover_image && "justify-center"}
```

Comment:
It's not safe to use && on classes, because the way && works is by returning the last value checked. So for example on an article that doesn't have image `article.cover_image && "justify-end"` actually evaluates to "null"
<img width="255" alt="image" src="https://github.com/user-attachments/assets/6935457b-f31c-499a-a12f-9858654e7361" />


```suggestion
          ${article.cover_image ? "justify-end" : "justify-center"}
```